openid-4-verifiable-presentations-1_0.md

```diff
@@ -2674,6 +2658,7 @@ The technology described in this specification was made available from contribut
 
    -22
 
+   * remove references to the openid4vci credential format section
```

Comment:
This should be in the -23 change history section (which isn't even showing up here, I suspect main needs to be merged into this PR first)

Reply:
merging main probably does solve this. we can merge with a note to move this to -23 later and I can fix it in a PR later. a lot of PRs now open need change log anyway

```suggestion
   * remove references to the openid4vci credential format section (needs to be moved to -23)
```

Reply:
It seemed easiest to just move it, I've done that now - approving!